### PR TITLE
bugfix/diktat-version-update

### LINKED ISSUE #210: We have released a new version of diktat - please update version of diktat in readme, usages and pom files
We will create automation process for it, but by for now you need to change in manually to 1.0.2

### PR DESCRIPTION
**Pull request by aktsay6**
### What's done:
  * Updated poms to version 1.0.3-SNAPSHOT
  * Readme and diktat.check-version updated to 1.0.2-SNAPSHOT

## Actions checklist
* [ ] Implemented Rule, added Warnings
* [ ] Added tests on checks
* [ ] Added tests on fixers
* [ ] Updated rules-config.json
* [ ] Updated available-rules.md

#Closes #210

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Then, add this plugin:
               <dependency>
                   <groupId>org.cqfn.diktat</groupId>
                   <artifactId>diktat-rules</artifactId>
-                  <version>1.0.1</version> <!-- replace it with diktat latest version -->
+                  <version>1.0.2-SNAPSHOT</version> <!-- replace it with diktat latest version -->
```
Comment (petertrr) [inline README.md:114]: The released version is `1.0.2`, the `snapshot` suffix is for development phase only.

File: pom.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
         <jackson.version>2.11.2</jackson.version>
         <slf4j.version>1.7.30</slf4j.version>
         <commons-cli.version>1.4</commons-cli.version>
-        <diktat-check.version>1.0.1</diktat-check.version>
+        <diktat-check.version>1.0.2-SNAPSHOT</diktat-check.version>
```
Comment (petertrr) [inline pom.xml:23]:
```suggestion
        <diktat-check.version>1.0.2</diktat-check.version>
```
Same here

File: README.md (hunk 1 above)
Comment (aktsay6) [reply README.md]: fixed

File: pom.xml (hunk 2 above)
Comment (aktsay6) [reply pom.xml]: fixed

File: README.md [hunk 3]
```diff
@@ -155,7 +155,8 @@ dependencies {
     }
 
     // diktat ruleset
-    ktlint("org.cqfn.diktat:diktat-rules:1.0.1") {
+    ktlint("org.cqfn.diktat:diktat-rules:1.0.3-SNAPSHOT") {
```
Comment (petertrr) [inline README.md:158]: This is a usage instruction for possible users, therefore it should use released version which can be obtained from maven repo. Versions with `SNAPSHOT` suffix are accessible only if you build from source. Also, this line is duplicated.

File: build.gradle.kts [hunk 4]
```diff
@@ -17,7 +17,7 @@ dependencies {
     }
 
     // diktat ruleset
-    ktlint("org.cqfn.diktat:diktat-rules:1.0.1") {
+    ktlint("org.cqfn.diktat:diktat-rules:1.0.3-SNAPSHOT") {
```
Comment (petertrr) [inline build.gradle.kts:20]:
```suggestion
    ktlint("org.cqfn.diktat:diktat-rules:1.0.2") {
```
Same here, it is and example on usage with released version.

File: README.md [hunk 5]
```diff
@@ -155,7 +155,8 @@ dependencies {
     }
 
     // diktat ruleset
-    ktlint("org.cqfn.diktat:diktat-rules:1.0.1") {
+    ktlint("org.cqfn.diktat:diktat-rules:1.0.2") {
+    ktlint("org.cqfn.diktat:diktat-rules:1.0.2") {
```
Comment (petertrr) [inline README.md:159]:
```suggestion
    ktlint("org.cqfn.diktat:diktat-rules:1.0.2") {
```

**Review by petertrr — APPROVED** [review]
lgtm

**Pull request merged**